.eslintrc.js

```diff
@@ -293,6 +309,8 @@ module.exports = {
               SAFER_LODASH_SET_HEADER,
               SAFER_LODASH_SET_LODASH_HEADER,
               SAFER_LODASH_SET_DEFINITELYTYPED_HEADER,
+              KBN_HANDLEBARS_HEADER,
```

Comment:
Do we need to have these licenses listed here? Will this header ever be used for files outside `packages/kbn-handlebars` ?

Reply:
You're right, it will not be used outside of the `packages/kbn-handlebars` directory. But I believe that's what's also stated here. This is a list of licenses _not_ allowed in files generally. Further down in the file you'll find a rule that only matches the `packages/kbn-handlebars` directory where the new license _is_ allowed.